crates/dbs-virtio-devices/src/device.rs

```diff
@@ -842,8 +842,8 @@ pub(crate) mod tests {
         let gm = GuestMemoryAtomic::<GuestMemoryMmap>::new(gmm);
 
         let queues = vec![
-            VirtioQueueConfig::create(0, 0).unwrap(),
-            VirtioQueueConfig::create(0, 0).unwrap(),
```

Comment:
What error will be reported if it is not modified?

Reply:
In the new Virtio-queue, when creating Virtio queue, it will check that the max size is a power of 2 because the valid queue sizes are a power of 2 as per the specification. 

So here we could not successfully create a Virtio queue